### ISSUE TITLE
Error in plugin 'run-sequence(build.html_css)', with:  npm start

### ISSUE THREAD

**Issue by webmobiles** [mgechev/angular-seed#1997]
using node  v7.10.0

```
[x ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see use [gitter](https://gitter.im/mgechev/angular2-seed) or [stackoverflow](https://stackoverflow.com/questions/tagged/angular2)
```

**Current behavior**
Error with:  npm start 

```
[17:05:44] 'build.html_css' errored after 3.06 s
[17:05:44] Error: ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
    at Object.fs.readdirSync (fs.js:914:18)
    at Object.getInstalledBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/extensions.js:124:13)
    at foundBinariesList (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:20:15)
    at foundBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:15:5)
    at Object.module.exports.missingBinary (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:45:5)
    at module.exports (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/binding.js:15:30)
    at Object.<anonymous> (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
[17:05:44] 'serve.dev' errored after 3.25 s
[17:05:44] Error in plugin 'run-sequence(build.html_css)'
Message:
    ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
Details:
    errno: -2
    code: ENOENT
    syscall: scandir
    path: /home/meridia/angular/mec2017native/node_modules/node-sass/vendor
Stack:
Error: ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
    at Object.fs.readdirSync (fs.js:914:18)
    at Object.getInstalledBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/extensions.js:124:13)
    at foundBinariesList (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:20:15)
    at foundBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:15:5)
    at Object.module.exports.missingBinary (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:45:5)
    at module.exports (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/binding.js:15:30)
    at Object.<anonymous> (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
[17:05:44] 'build.dev' errored after 3.25 s
[17:05:44] Error in plugin 'run-sequence(serve.dev)'
Message:
    ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
Details:
    errno: -2
    code: ENOENT
    syscall: scandir
    path: /home/meridia/angular/mec2017native/node_modules/node-sass/vendor
Stack:
Error: ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
    at Object.fs.readdirSync (fs.js:914:18)
    at Object.getInstalledBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/extensions.js:124:13)
    at foundBinariesList (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:20:15)
    at foundBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:15:5)
    at Object.module.exports.missingBinary (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:45:5)
    at module.exports (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/binding.js:15:30)
    at Object.<anonymous> (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
[17:05:44] 'build.dev' errored after 3.25 s
[17:05:44] Error in plugin 'run-sequence(build.html_css)'
Message:
    ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
Details:
    errno: -2
    code: ENOENT
    syscall: scandir
    path: /home/meridia/angular/mec2017native/node_modules/node-sass/vendor
Stack:
Error: ENOENT: no such file or directory, scandir '/home/meridia/angular/mec2017native/node_modules/node-sass/vendor'
    at Object.fs.readdirSync (fs.js:914:18)
    at Object.getInstalledBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/extensions.js:124:13)
    at foundBinariesList (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:20:15)
    at foundBinaries (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:15:5)
    at Object.module.exports.missingBinary (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/errors.js:45:5)
    at module.exports (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/binding.js:15:30)
    at Object.<anonymous> (/home/meridia/angular/mec2017native/node_modules/node-sass/lib/index.js:14:35)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! angular-seed-advanced@0.0.0 start: `gulp serve.dev --color`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the angular-seed-advanced@0.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/meridia/.npm/_logs/2017-06-29T15_05_45_056Z-debug.log
```

**Comment by MaverickW** [comment]
It's know issue of yarn package manager: https://github.com/sass/node-sass/issues/1804.

**TL;DR;**
The next thing should help you:
`npm rebuild node-sass `
